cockroachdb/manifests-cert-manager/statefulset.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   annotations:
     "app.uw.systems/description": "Used to store data projections of events in multiple services."
     "app.uw.systems/tier": "tier_1"
```

Comment:
Why is this tier1? I feel like we had a recent discussion about how to tier manifests from a base repo like this (where the tier should really be up to the user) but I forget what we concluded

Reply:
[Moved the annotation to examples](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/44/commits/3b28b3016d7186996e14bd1c85aa00cb8e06eed8). 
I agree, we should not assume something is important, especially considering [the documented weight of tier_1](https://wiki.uw.systems/posts/ops-level-nz4v4ka0#h1u0u-app-uw-systems-tier)